src/ProductConstructionService/ProductConstructionService.BarViz/Components/ChannelNavMenuItem.razor

```diff
@@ -0,0 +1,41 @@
+﻿@using ProductConstructionService.Client
+@using ProductConstructionService.Client.Models;
+@using System.Collections.Immutable
+@inject IProductConstructionServiceApi PcsApi
+
+<FluentNavGroup
+    Href=@($"channel/{Channel.Id}")
+    Icon="@(new Icons.Regular.Size20.Channel())"
+    IconColor="Color.Accent"
+    Title="@Channel.Name"
+    ExpandedChanged="OnExpandedChanged">
+        @if (loaded)
+        {
+            @foreach(var repo in repositories)
+            {
+                <FluentNavLink Icon="@(new Icons.Regular.Size20.DocumentSave())" IconColor="Color.Accent" Href="@($"{Channel.Id}/{Uri.EscapeDataString(repo)}/latest/graph")">@(new Uri(repo).Segments.Last())</FluentNavLink>
```

Comment:
Shouldn't the start of the href be `channel/{Channel.Id}/...` to be consistent with the channel route?

Reply:
For now, I just copied what is in the real Maestro and that points to this. E.g.

https://maestro.dot.net/2914/https:%2F%2Fdev.azure.com%2Fdevdiv%2FDevDiv%2F_git%2Fvs-code-coverage/latest/graph